chirp/drivers/uv5r.py

```diff
@@ -193,8 +193,10 @@
      txpower3:2;
 } vfob;
 
-#seekto 0x0F56;
-u16 fm_presets;
+#seekto 0x0F48;     // some new radios do not support this feature
+char bcstfmlo[14];  // broadcast FM 65-75 MHz band
+u16 fm_presets;     // broadcast FM frequency
+char bcstfmhi[14];  // broadcast FM 75-108 MHz band
```

Comment:
Oh I see you added these elements here just now, I got it.

I guess maybe these could be different in some radios for other geographies maybe? I guess

Reply:
Yes. I only have 1 radio here with the issue. But the 2 or 3 images that I have gotten from others all have bcstfmlo[14] and bcstfmhi[14] fully populated with 0xFF. So far I have never seen an 0xFF in an older "good" radio so I just check for a single 0xFF when uploading. I would rather err on the side of not uploading to a "good" radio than accidentally upload to a "bad" one.